fix path to stereo_chemical_props.txt when AlphaFold is installed with EasyBuild
author: Kenneth Hoste (HPC-UGent)
update 2.0.0 -> 2.1.0: Thomas Hoffmann (EMBL Heidelberg)
--- alphafold/common/residue_constants.py.orig	2023-04-17 13:59:57.535153646 +0200
+++ alphafold/common/residue_constants.py	2023-04-17 14:02:33.555747657 +0200
@@ -25,7 +25,7 @@
 # Internal import (35fd).
 
 from . import __file__ 
-stereo_chemical_props_path = os.path.join(os.path.dirname(__file__), f'stereo_chemical_props.txt')  
+stereo_chemical_props_path = os.path.join(os.getenv('EBROOTALPHAFOLD', 'alphafold/common'), 'stereo_chemical_props.txt')
 
 # Distance from one CA to next CA [trans configuration: omega = 180].
 ca_ca = 3.80209737096
